spec/importers/mahonia_mapper_spec.rb

```diff
@@ -103,5 +100,29 @@
 
       expect(mapper.map_field(:title)).to eq(["One"])
     end
+
+    it "provides a creator field that returns first and last author name" do
+      mapper.metadata = { "author1_fname" => "Marie", "author1_lname" => "Curie" }
+
+      expect(mapper.creator).to eq(["Marie Curie"])
+    end
+
+    it "won't map creator if author1_fname and author1_lname are both nil" do
+      mapper.metadata = { "author1_fname" => nil, "author1_lname" => nil }
+
+      expect(mapper.creator.nil?).to be_truthy
+    end
+
+    it "will map creator if only author1_lname is nil" do
+      mapper.metadata = { "author1_fname" => "Marie", "author1_lname" => nil }
+
+      expect(mapper.creator).to eq(["Marie"])
+    end
+
+    it "provides 10 creators from 10 author first and last name pairs" do
+      mapper.metadata = authors
+
+      expect(mapper.creator).to eq(ten_creators)
```

Comment:
Maybe `contain_exactly`? I'm not sure we care to guarantee the order.

Reply:
Yeah, good point.

Reply:
I think this is working out okay as it is, actually.